README.md

```diff
@@ -1,20 +1,33 @@
-HumHub - Social Network Kit
+HumHub - Putting People and Pieces together
 ===========================
 
 [![Test Status](https://github.com/humhub/humhub/workflows/PHP%20Codeception%20Tests/badge.svg)](https://github.com/humhub/humhub/actions)
 [![Yii2](https://img.shields.io/badge/Powered_by-Yii_Framework-green.svg?style=flat)](http://www.yiiframework.com/)
 [![CLA assistant](https://cla-assistant.io/readme/badge/humhub/humhub)](https://cla-assistant.io/humhub/humhub)
 
-HumHub is a feature rich and highly flexible OpenSource Social Network Kit written in PHP.
+**HumHub is an intuitive to use and modular designed open-source software**, used primarily as social network, knowledge database, intranet or information and communication platform.
 
-It's perfect for individual:
-- Social Intranets
-- Enterprise Social Networks
-- Private Social Networks
+**The software is written in PHP** and is best described by dividing into 4 main parts: **User, Spaces, Content and Modules.**
 
-More information:
-- [Homepage & Demo](http://www.humhub.org)
-- [Documentation & Class Reference](http://docs.humhub.org)
-- [Community](http://community.humhub.com/)
-- [Licence](https://www.humhub.com/licences)
+**User:** All users have their own customisable profile (including name, profile picture, cover photo and personal information) and can follow and interact with each other. If wished and enabled, users can create own content, comment posts and join Spaces. Profile fields, permissions and all settings can be defined individually by the network operator (administrator).
```

Comment:
Should also mention that the friendship function can also be enabled as well, so around `follow` something should be added here to mention friendships.

Reply:
There are so many things and functions that could be mentioned. All in all the text should be short and give a good overview of HumHub and it´s functions. But thanks for the feedback, will take a look into it once again soon.